go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -172,20 +172,15 @@ func (bh *S3BackupHandle) AddFile(ctx context.Context, filename string, filesize
 			u.PartSize = partSizeBytes
 		})
 		object := objName(bh.dir, bh.name, filename)
-		sendStats := bh.bs.params.Stats.Scope(stats.Operation("AWS:Request:Send"))
-		_, err := uploader.UploadWithContext(ctx, &s3manager.UploadInput{
+		_, err := uploader.Upload(&s3manager.UploadInput{
 			Bucket:               &bucket,
 			Key:                  object,
 			Body:                 reader,
 			ServerSideEncryption: bh.bs.s3SSE.awsAlg,
 			SSECustomerAlgorithm: bh.bs.s3SSE.customerAlg,
 			SSECustomerKey:       bh.bs.s3SSE.customerKey,
 			SSECustomerKeyMD5:    bh.bs.s3SSE.customerMd5,
-		}, s3manager.WithUploaderRequestOptions(func(r *request.Request) {
```

Comment:
Hey @L3o-pold have you tried changing `UploadWithContext` to `Upload` without also deleting these lines 184-188? Ideally we would be able to work around the issue you are having by switching to `Upload`, without turning of stats collection for this area of code.